src/main/java/org/retest/rebazer/domain/PullRequest.java

```diff
@@ -2,16 +2,16 @@
 
 import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 
 @Data
 @Builder
 public class PullRequest {
 
 	private final Integer id;
-	private final String repo;
 	private final String source;
 	private final String destination;
-	private final String url;
+	@EqualsAndHashCode.Exclude
```

Comment:
Should this be tested 😛 ?

Reply:
There is a lot of code which should be tested, but in separate branches to close this one finally. I made a issue for that https://github.com/retest/rebazer/issues/39